### PR TITLE
add emissions topics

### PR DESCRIPTION
**Pull request by n-h-diaz**

### CODE REVIEW

**Review by pradh — COMMENTED** [review]
Thanks!

File: server/lib/nl/topic.py [hunk 1]
```diff
@@ -188,6 +195,51 @@
         "dc/0gettc3bc60cb",  # Drove alone in car/truck/van
         "dc/vt2q292eme79f",  # Taxicab/Motorcycle/Bicycle/etc
     ],
+    "dc/svpg/GreenhouseGasEmissions": [
+        "Annual_Emissions_GreenhouseGas_Agriculture",
+        "Annual_Emissions_GreenhouseGas_FuelCombustionInBuildings",
+        "Annual_Emissions_GreenhouseGas_ElectricityGeneration",
+        "Annual_Emissions_GreenhouseGas_ForestryAndLandUse",
+        "Annual_Emissions_GreenhouseGas_Manufacturing",
+        "Annual_Emissions_GreenhouseGas_MaritimeTransport",
```
[the remark this answers is not in the document]
Comment (n-h-diaz) [reply server/lib/nl/topic.py:204]: the sectors are different

File: server/lib/nl/topic.py [hunk 2]
```diff
@@ -211,6 +263,16 @@
         "Houshold Income",
     "dc/svpg/WorkCommutePeerGroup":
         "Modes of Commute",
+    "dc/svpg/GreenhouseGasEmissions":
+        "Greenhouse Gas Emissions",
```
[the remark this answers is not in the document]
Comment (n-h-diaz) [reply server/lib/nl/topic.py]: done

File: server/lib/nl/topic.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (n-h-diaz) [reply server/lib/nl/topic.py]: removed maritime transport

File: server/lib/nl/topic.py [hunk 3]
```diff
@@ -188,6 +195,51 @@
...
+        "Annual_Emissions_GreenhouseGas_Transportation",
+        "Annual_Emissions_GreenhouseGas_WasteManagement",
+    ],
+    "dc/svpg/CarbonDioxideEmissions": [
+        "Annual_Emissions_CarbonDioxide_FossilFuelOperations",
+        "Annual_Emissions_CarbonDioxide_Power",
+        "Annual_Emissions_CarbonDioxide_WasteManagement",
+        "Annual_Emissions_CarbonDioxide_Agriculture",
+        "Annual_Emissions_CarbonDioxide_MineralExtraction",
+        "Annual_Emissions_CarbonDioxide_FluorinatedGases",
```
[the remark this answers is not in the document]
Comment (n-h-diaz) [reply server/lib/nl/topic.py]: I removed the non-overlapping ones, so now each has 7 sectors

[the remark this answers is not in the document]
Comment (pradh) [reply server/lib/nl/topic.py]: BTW, if you'd like add all non-overlapping ones, and we can pick the top ones.

**Review by pradh — APPROVED** [review]
Thanks!

**Pull request merged**